class E2EError(Exception):
    pass
